08-rails/reference/.rails-template.rb

```diff
@@ -114,6 +137,28 @@
     RUBY
   end
 
+   gsub_file "test/test_helper.rb", 'parallelize(workers: :number_of_processors)' do
+    "# parallelize(workers: :number_of_processors) # causes out of order output."
```

Comment:
If I can remember how I implemented a parallel test runner with ordered output for RSpec at Socrata I might contribute a bug fix because running in parallel is nice.  (You had to manually opt in there though, so the technique might not work here.)